README.md

```diff
@@ -49,12 +54,12 @@ A: If you don't know how to build a project, please wait for a new beta or relea
 
 Q: When is the official release?
 
-A: It is currently released! Come download it on our <a href="https://discord.gg/RNyRgtv">Discord</a>!
+A: It is currently released!  Come download it on our <a href="https://discord.gg/RNyRgtv">Discord</a>!
 
 
 Q: Can I use Hyperium now?
 
-A: Beta versions, yes! Join the [Discord](https://discord.gg/RNyRgtv) or visit our [website](https://hyperium.cc)!
+A: Beta versions, yes!  [Click me](#beta-test-the-client) for more info.  
```

Comment:
what is #beta-test-the-client meant to do lol

Reply:
It scrolls up to the new section I added.  Its an HTML link thing.

Reply:
clients not even in beta anymore